src/FNA3D_Driver_D3D11.c

```diff
@@ -4582,6 +4586,11 @@ static uint8_t D3D11_SupportsNoOverwrite(FNA3D_Renderer *driverData)
 	return 1;
 }
 
+static uint8_t D3D11_SupportsSRGBFrameBuffer(FNA3D_Renderer *driverData)
+{
+	return 1;
```

Comment:
Should probably do ID3D11Device_CheckFormatSupport, similar to S3TC (which I just noticed uses the return value incorrectly...)